{"links": {"self": {"href": "data/repositories/osrf/sdformat/pullrequests/450/comments/70717300.json"}, "html": {"href": "#!/osrf/sdformat/pull-requests/450/_/diff#comment-70717300"}}, "deleted": false, "pullrequest": {"type": "pullrequest", "id": 450, "links": {"self": {"href": "data/repositories/osrf/sdformat/pullrequests/450.json"}, "html": {"href": "#!/osrf/sdformat/pull-requests/450"}}, "title": "Added missing Element accessors"}, "content": {"raw": "The one question I have with all of this is whether we should throw an exception if the `Element` method is called before `Load` \\(rather than the current practice of throwing nullptr\\).  It\u2019s a bit more code, but if we really expect it to be an \u201cexceptional\u201d situation, then we should probably do that.  Thoughts?", "markup": "markdown", "html": "<p>The one question I have with all of this is whether we should throw an exception if the <code>Element</code> method is called before <code>Load</code> (rather than the current practice of throwing nullptr).  It\u2019s a bit more code, but if we really expect it to be an \u201cexceptional\u201d situation, then we should probably do that.  Thoughts?</p>", "type": "rendered"}, "created_on": "2018-07-23T12:45:56.961402+00:00", "user": {"display_name": "clalancette", "uuid": "{02b311e1-297d-4232-89c4-5485d0a02911}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B02b311e1-297d-4232-89c4-5485d0a02911%7D"}, "html": {"href": "https://bitbucket.org/%7B02b311e1-297d-4232-89c4-5485d0a02911%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:ad6ed80b-ec66-45cb-9504-b1094a415b7f/048a0933-60ac-4910-a6a5-ed058bdea8eb/128"}}, "nickname": "clalancette", "type": "user", "account_id": "557058:ad6ed80b-ec66-45cb-9504-b1094a415b7f"}, "updated_on": "2018-07-23T12:45:56.971862+00:00", "type": "pullrequest_comment", "id": 70717300}